scripts/emulator-import-export-tests/tests.ts

```diff
@@ -0,0 +1,559 @@
+import { expect } from "chai";
```

Comment:
FYI, the code in this file used to be part of triggers-end-to-end-test, and was moved over basically unchanged.